app/services/idv/attempter.rb

```diff
@@ -9,14 +9,17 @@ def initialize(current_user)
     end
 
     def increment
-      current_user.update!(
-        idv_attempts: current_user.idv_attempts + 1,
-        idv_attempted_at: Time.zone.now
-      )
+      UpdateUser.new(
+        user: current_user,
+        attributes: { idv_attempts: current_user.idv_attempts + 1, idv_attempted_at: Time.current }
+      ).call
```

Comment:
honestly: line length. I saw that we are using them both so thought I'd take the short cut. Seems like usage is kind of 50/50?